doc/rtd/explanation/format.rst

```diff
@@ -141,66 +225,128 @@ Create user data containing 3 shell scripts:
 
     $ cloud-init devel make-mime -a always.sh:x-shellscript-per-boot -a instance.sh:x-shellscript-per-instance -a once.sh:x-shellscript-per-once
 
-``include`` file
-================
 
-This content is an :file:`include` file.
+Cloud config archive
+====================
+
+| **Header:** #cloud-config-archive
+| **Content-Type:** text/cloud-config-archive
+
+Example
+-------
+
+.. code-block:: shell
+
+    #cloud-config-archive
+    - type: "text/cloud-boothook"
+    content: |
+        #!/bin/sh
+        echo "this is from a boothook." > /var/tmp/boothook.txt
+    - type: "text/cloud-config"
+    content: |
+        bootcmd:
+        - echo "this is from a cloud-config." > /var/tmp/bootcmd.txt
+
+Explanation
+-----------
+
+A cloud-config-archive is a way to specify more than one type of data
+using YAML. It can be seen as an alternative to building a MIME multi-part
+archive manually.
+
+The format is a list of dictionaries.
 
-The file contains a list of URLs, one per line. Each of the URLs will be read
-and their content will be passed through this same set of rules, i.e., the
-content read from the URL can be gzipped, MIME multi-part, or plain text. If
-an error occurs reading a file the remaining files will not be read.
+Required fields:
 
-Begins with: ``#include`` or ``Content-Type: text/x-include-url``  when using
-a MIME archive.
+* ``type``: The content type of the MIME part
```

Comment:
This format doesn't use a MIME part, does it? Maybe something like this instead?

> type: The Content-Type identifier for the type of user data in content
> content: The user data configuration

Reply:
It translates fairly directly to a mime part that gets generated from the config. I can pull the MIME details out of this section, but IMO it'll make some of the other fields harder to understand.